src/backend/distributed/transaction/lock_graph.c

```diff
@@ -447,6 +448,18 @@ BuildLocalWaitGraph(void)
 			continue;
 		}
 
+		/*
+		 * We should skip if the process blocked on the relation extension since
+		 * those locks are hold for a short duration while the relation is actually
+		 * extended on the disk and released as soon as the extension is done. Thus,
```

Comment:
I'd move a bunch of this down in the function comment, so it's easy to find.

Reply:
Makes sense!